pkg/controller/replication/replication_controller.go

```diff
@@ -660,13 +663,13 @@ func (rm *ReplicationManager) syncReplicationController(key string) error {
 			return aggregate
 		}
 	} else {
-		podList, err := rm.podStore.Pods(rc.Namespace).List(labels.Set(rc.Spec.Selector).AsSelector())
+		pods, err := rm.podStore.Pods(rc.Namespace).List(labels.Set(rc.Spec.Selector).AsSelector())
```

Comment:
This is the code where GC is not enabled, so the problem is there even without GC.